test/autograd/test_load_derivatives.py

```diff
@@ -0,0 +1,62 @@
+import dataclasses
+import unittest
+
+from tools.autograd import load_derivatives
+import tools.codegen.model
+
+class TestCreateDerivative(unittest.TestCase):
+
+    def test_this_is_run(self):
```

Comment:
For the time being, I put these test cases in tests/test_autograd.py.

Trying to separate that file out atomically is challenging, but I have a few ideas for how to do it incrementally. I'll update https://github.com/pytorch/pytorch/issues/63085 with my thoughts.

Reply:
#63400 starts the process of splitting up test_autograd. Since it requires some other infrastructure to make it work, I'm treating that as a separate chain of work.